### PR TITLE
feat(equipment): TEMS-101 add equipment type enum

### PR DESCRIPTION
**Pull request by princhcanal**
## Description

- converts equipment type from `string` to `EquipmentTypeEnum`
- removes `cy.wait(numValue)` calls

## Checklist

_First, create a draft pull request. Then mark the PR as ready for review when all necessary checkbox items has been completed_

- [x] Reviewed own code
- [x] Commented on code that is hard to understand
- [x] Implemented tests for the feature/bugfix
- [x] All **_Required_** GitHub status checks pass
- [x] The frontend and backend PR previews have been deployed

**Backend only**

- [x] Added test data for new entity
- [x] Generated the client api if there are new or deleted endpoints and/or DTOs

### CODE REVIEW

File: backend/prisma/migrations/20221106015253_add_equipment_type/migration.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+-- CreateEnum
+CREATE TYPE "EquipmentType" AS ENUM ('PHOTO_DOCUMENTATION', 'VIDEO_DOCUMENTATION', 'PHOTO_AND_VIDEO_DOCUMENTATION', 'LIVE_STREAMING', 'MUSIC_BAND', 'OTHERS');
+
+UPDATE "Equipment"
+  SET "type" = 'OTHERS';
```
Comment (princhcanal) [inline backend/prisma/migrations/20221106015253_add_equipment_type/migration.sql:5]: @noice-noise for existing equipments, i set the type to `OTHERS`. let me know if i should change it :)

File: backend/src/venue/venue.service.ts [hunk 2]
```diff
@@ -29,6 +34,14 @@ export class VenueService {
       const venue = await this.prisma.venues.create({
         data,
       });
+
+      const equipmentDTO: EquipmentDTO = {
+        name: `${data.name} ${EQUIPMENT_SET_POSTFIX}`,
+        type: EquipmentTypeEnum.OTHERS,
```
Comment (princhcanal) [inline backend/src/venue/venue.service.ts:40]: @noice-noise also for equipments that created from creating venues

**Comment by princhcanal** [comment]
a lot of the event e2e tests are failing but i can have a look at that later

**Review by beefysalad — APPROVED** [review]
LGTM! :KEKEZ:

**Pull request merged**